examples/Server_With_All_Interfaces-Settings.js

```diff
@@ -19,6 +19,7 @@ var moscaSetting = {
         { type: "https", port: 3001, bundle: true, credentials: { keyPath: SECURE_KEY, certPath: SECURE_CERT } }
     ],
     stats: false,
+    qos2Puback: false, // can set to true if using a client which will eat puback for QOS 2; e.g. mqtt.js
```

Comment:
I move the 'if' so it checks the QOS before anything else anyway.  It's only here as an option to protect stupids like me who set mqtt.js to QoS2 :).  (well node-red does default to QoS2 for mqtt subscription; fooled me).